etc/kayobe/kolla.yml

```diff
@@ -69,7 +69,7 @@ kolla_ansible_source_version: "{{ stackhpc_kolla_ansible_source_version }}"
 # Kolla base container image distribution. Options are "centos", "debian",
 # "rocky", "ubuntu". Default is
 # {{ 'centos' if (os_distribution == 'rocky' and os_release == '8') else os_distribution }}.
-#kolla_base_distro:
+kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}"
```

Comment:
This will cause us to always build container images with the same distro as the seed. You can move this override to etc/kayobe/kolla/globals.yml instead, so that it's only used by k-a

Reply:
Done